cgi-bin/DW/External/User.pm

```diff
@@ -69,13 +71,17 @@ sub ljuser_display {
     my $display_class = $opts{no_ljuser_class} ? "" : " class='ljuser'";
     my $domain = $self->site->{domain} ? $self->site->{domain} : $self->site->{hostname};
 
+    $nolink = 1 if (first {$domain eq $_} @DW::External::Site::deadsites);
```

Comment:
I appreciate that you went on a journey 😂 but it's much more typical to use a hash than an array here:
```
my %deadsites;
...
$nolink = 1 if exists $deadsites{$domain};
```
Let us know if you need a more detailed explanation!